prov/gni/src/gnix_av.c

```diff
@@ -172,6 +173,10 @@ static int table_insert(struct gnix_fid_av *av_priv, const void *addr,
 		/* check if this ep_name fits in the av context bits */
 		if (temp->name_type & GNIX_EPN_TYPE_SEP) {
 			if ((1 << av_priv->rx_ctx_bits) < temp->rx_ctx_cnt) {
+				if (flags && FI_SYNC_ERR) {
+					entry_err[i] = -FI_EINVAL;
+					fi_addr[i] = FI_ADDR_NOTAVAIL;
+				}
```

Comment:
Is it supposed to return on first error?  Or should it try the rest and fill in the array?